### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by jmininger**

### CODE REVIEW

File: Experimental-ckb-cli.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+the message as hex, instead of ascii characters
+```
+CKB> util sign-message --binary-hex '0x61' --from-account <my-ledger-account> 
+```
+
+### Signing a Hash ###
+The ledger also allows for the signing of a pre-hashed message. Due to the potential security risks of this option, a user must "opt-in" to this
+setting by going into the `Configuration` section of the app, and setting `Allow sign hash` to `On`.
+
+Note that the current implementation of the `ledger-sign-hash` util has a verification system built-in, and, as such, only accepts
```
Comment (mikereinhart) [inline Experimental-ckb-cli.md:47]: Can we expand on this point to mention why the existing `verify-signature` command will not work?

File: Experimental-ckb-cli.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+blake2b hashes that use the Nervos personalization. A user can use `util blake2b` to get a hash of this sort. The ledger itself does not have
+this constraint.
+```
+CKB> util ledger-sign-hash --binary-hex <message hash> --from-account <my-ledger-account> --path <optional derivation path>
```
Comment (mikereinhart) [inline Experimental-ckb-cli.md:51]: Just confirming `--path` is actually optional, I think I remember it being required

File: README.md [hunk 3]
```diff
@@ -406,6 +420,10 @@ total_count: 1
 ```
 
 ## Message Signing ##
+*Note that the following instructions reflect an anticipation of the future state of ckb-cli and are not all
+compatible with Obsidian's current ledger-app branch. For more instructions on using the ledger's message signing
+features with the ledger-app branch of ckb-cli see [](Experimental-ckb-cli.md) *
```
Comment (mikereinhart) [inline README.md:425]: The link at the end of this sentence needs fixing

File: README.md [hunk 4]
```diff
@@ -240,14 +240,28 @@ The Nervos Ledger app is built to work with CKB-CLI. Some of these CKB-CLI subco
...
 All commands that follow prefixed with ‘CKB>’ should be run in the prompt provided by the above command.
 
+### Installing the client manually:
+It is also possible to install ckb-cli without nix. Should you choose to do so, please
+follow the upstream repo's instructions regarding the installation of dependencies
+```
+git clone https://github.com/obsidiansystems/ckb-cli.git
+cd ckb-cli
+git checkout ledger-app
+cargo run
```
Comment (mikereinhart) [inline README.md:262]: The full upstream command is `cargo install --path . -f` not `cargo run`

File: Experimental-ckb-cli.md (hunk 2 above)
Comment (jmininger) [reply Experimental-ckb-cli.md:51]: I think there was a bug i had to fix, but it works now.

File: README.md (hunk 4 above)
Comment (jmininger) [reply README.md:262]: `cargo run` will run the app. `cargo install` will, i imagine, put it on path. Ill add it in

**Pull request merged**